contribution/AniketNS.mdx

```diff
@@ -0,0 +1,85 @@
+        </svg>
+      }
+      level="Newbie" 
+    >
+        your experience
+    </Skill>
+    <Skill
+      name="skill name"
+      href="official website"
+      level="Intermidiate"
+      icon={
+      add a svg icon of your skills
+      }
+    >
+   your experience
+    </Skill>
```

Comment:
add your details such as skills name, experience etc.

Reply:
I don't have any experience. Like I'm a college student.

Reply:
> I don't have any experience. Like I'm a college student.

then leave it blank or add `learning`

Reply:
yup, I've removed it. Is that Ok?